components/tabs/index.css

```diff
@@ -96,6 +96,9 @@ governing permissions and limitations under the License.
   font-size: var(--spectrum-tabs-text-size);
   font-weight: var(--spectrum-tabs-text-font-weight);
 
+  /* support links */
+  text-decoration: none;
```

Comment:
I think you also need a `color: inherit` or similar to support anchor tags with `href` attributes.